src/plugins/mutualGroupDMs/index.tsx

```diff
@@ -47,7 +47,7 @@ export default definePlugin({
         {
             find: ".Messages.USER_PROFILE_MODAL", // Note: the module is lazy-loaded
             replacement: {
-                match: /(?<=\.MUTUAL_GUILDS\}\),)(?=(\i\.bot).{0,20}(\(0,\i\.jsx\)\(.{0,100}id:))/,
+                match: /(?<=\.MUTUAL_GUILDS,children:\i}\),)(?=(\i\.bot).{0,20}(\(0,\i\.jsx\)\(.{0,100}id:))/,
                 replace: '($1||arguments[0].isCurrentUser)?null:$2"MUTUAL_GDMS",children:"Mutual Groups"}),'
```

Comment:
```suggestion
                match: /(?<=\.tabBarItem.{0,50}MUTUAL_GUILDS.+?}\),)(?=.+?(\(0,\i\.jsxs?\)\(.{0,100}id:))/,
                replace: '(arguments[0].user.bot||arguments[0].isCurrentUser)?null:$1"MUTUAL_GDMS",children:"Mutual Groups"}),'
```

This works on both stable and canary